src/content/en/2021/security.md

```diff
@@ -1099,54 +1104,58 @@ We now turn to cryptominer market share in the following figure.
 
 {{ figure_markup(
   image="security-usage-of-well-known-security.png",
   caption="Use of `security.txt`.",
-  description="Bar chart showing 4.93% of websites in desktop and 4.89% of websites in mobile have `security.txt` endpoint.",
+  description="Bar chart showing 4.9% of websites in desktop and 4.9% of websites in mobile have `security.txt` endpoint.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=1958147881&format=interactive",
   sheets_gid="399976976",
   sql_file="well-known_security.sql"
   )
 }}
 
-And the following figure shows that the policy is the most used property of utilized `security.txt` files.
+We see that just under 5% of the websites already use this standard—even though it is still a very young standard. We hope to see this number increase in future, since having a `security.txt` is an indication that the website administrators care about security and are willing to work with vulnerability reporters to improve security of their website.
 
 {{ figure_markup(
   image="security-usage-of-properties-in-well-known-security.png",
   caption="Use of security.txt properties.",
-  description="Bar chart showing use of different properties in security.txt. `signed` is 0.37% in desktop and 0.28% in mobile, Canonical is 2.75% in desktop and 2.72% in mobile, Encryption is 2.55% in desktop and 2.13% in mobile, Expires is 0.93% in desktop and 0.70% in mobile, Policy is 6.54$ in desktop and 6.39% in mobile.",
+  description="Bar chart showing use of different properties in security.txt. `signed` is 0.4% in desktop and 0.3% in mobile, Canonical is 2.8% in desktop and 2.7% in mobile, Encryption is 2.6% in desktop and 2.1% in mobile, Expires is 0.9% in desktop and 0.7% in mobile, Policy is 6.5% in desktop and 6.4% in mobile.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=644563241&format=interactive",
   sheets_gid="399976976",
   sql_file="well-known_security.sql"
   )
 }}
 
+We see that `Policy` is the most used property in the `security.txt` files. This property includes a link to the vulnerability disclosure policy for the website that helps researchers understand the reporting practices they need to follow. The other most used properties include `Canonical` and `Encryption`. `Canonical` is used to indicate where the `security.txt` file is located. If the URI used to retrieve the `security.txt` file doesn't match the list URIs in the `Canonical` fields, then the contents of the file should not be trusted. `Encryption` provides the security researchers with an encryption key that they can use for encrypted communication.
```

Comment:
```suggestion
We see that `Policy` is the most used property in the `security.txt` files, but even then it's only used in 6.4% of sites with a `security.txt` URL. This property includes a link to the vulnerability disclosure policy for the website that helps researchers understand the reporting practices they need to follow. This is therefore likely a better indicator of the real usage of `security.txt` since most file are expected to have a `Policy` value, meaning likely closer to 0.3% of all sites have a "real" `security.txt` file, rather than the 5% measured above.

Another interesting point is that when we look at just this subset of "real" security.txt URLs, Tumblr makes up 63%-65% of the usage. It looks like this is set by default for these domains to the Tumblr contact details. This is great on one hand to show how a single platform can drive adoption of these new security features, but on the other hand indicates a further reduction in actual site usage.

The other most used properties include `Canonical` and `Encryption`. `Canonical` is used to indicate where the `security.txt` file is located. If the URI used to retrieve the `security.txt` file doesn't match the list URIs in the `Canonical` fields, then the contents of the file should not be trusted. `Encryption` provides the security researchers with an encryption key that they can use for encrypted communication.
```